src/cmd/link.ts

```diff
@@ -1,21 +1,23 @@
 import path = require('path')
-import link, {linkFromGlobal, linkToGlobal} from '../api/link'
-import {PnpmOptions} from '../types'
+import link, { linkFromGlobal, linkToGlobal } from '../api/link'
+import { PnpmOptions } from '../types'
 
-export default (input: string[], opts: PnpmOptions & {globalPrefix: string}) => {
+export default (input: string[], opts: PnpmOptions & { globalPrefix: string }) => {
   const cwd = opts && opts.prefix || process.cwd()
 
   // pnpm link
   if (!input || !input.length) {
     return linkToGlobal(cwd, opts)
   }
 
-  // pnpm link ../foo
-  if (input[0].indexOf('.') === 0) {
-    const linkFrom = path.join(cwd, input[0])
-    return link(linkFrom, cwd, opts)
-  }
+  return Promise.all(input.map((inp: string) => {
```

Comment:
I have doubts this will work because we lock the global store. So you'll have to do this sequentially, wait for the previous link to finish